src/App.js

```diff
@@ -1,16 +1,35 @@
 import "./App.scss";
 import Header from "./components/Header/Header";
 import Footer from "./components/Footer/Footer";
+// import FakeWeatherData from "./data/FakeWeather.json";
+
 
-//configs
 const siteTitle = process.env.REACT_APP_SITE_TITLE ?? "CYF Weather";
 
 function App() {
+
   return (
     <div className="App">
       <Header title={siteTitle} />
+
       <main className="c-site-main" tabIndex="0">
+        {/* <div>
+          <img
+            className="weather-img"
+            src="https://cdn2.iconfinder.com/data/icons/weather-flat-14/64/weather02-512.png"
+            alt="Current Weather"
+          />
+        </div> */}
+
+
+        {/* {FakeWeatherData.list.map((data) => (
+          <div key={data.dt}>Temp : {data && data.main.temp}</div>
+        ))} */}
+
+        {/* <div>{location}</div> */}
+        {/* <div>{temp && temp.list[0].main.temp}</div> */}
```

Comment:
This is the main wrapper of all your other components. It should look something like this:
```
<Header />
<main>
   <CurrentWeather />
   <FutureWeather />
</main>
```

You currently have all the weather icon and weather data being displayed inside your Header component. This needs to be _fetched_ in the Header component (as you have done), but it needs to be _displayed_ inside of `<main>`, in a specific component (something like `CurrentWeather`).

Reply:
Let's get this merged in though and I'll create a ticket to move this logic to the right place in a different PR @zkhing

Reply:
I have improved some changes Lucy but I am stuck and it is still incomplete yet. Please have a look at the changes that I've made and suggest me how to improve when you have time. I do not know how to slice data from API for current and future weather. Many thanks